index.php

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang = "en">
+<!--Jacobi Williams
+        assignment6.htm
+        INFO2340.WW
+        Prof.Thoendel 
+        01/26/20 -->
+<head>
+<title>assignment6</title>
+<meta charset = "utf-8">
+</head>
+<body>
+    <h1>Hello, world!</h1>
+    <div> Allow me to introduce myself! My name is<?php echo "Jacobi Williams"; ?></div>
```

Comment:
solid php statments! complete with quotation marks and those pesky semicolons! 👍